microsoft/ebpf-for-windows
# Add Tracelogging (EbpfCore), reformat code to LLVM standards (#1565)

## Description

This PR adds tracing support to the ```ebpfcore``` driver. 

Along the way, I also took the liberty of streamlining the code to bring it in line with LLVM coding style.  I also performed some trivial code re-org. Specifically:
- move caller-callee functions in closer proximity
- ensure (struct) variable declaration, its initialization and subsequent use etc. are as contiguous as possible
- remove excessive blank lines, redundant code, declarations etc.
- ensure consistent single-line comment style (per LLVM coding style)

## Testing

- Ensure a clean full 'rebuild' of the entire project.
- Deploy ```ebpfcore``` driver to the test device, enable tracing, and ensure traces from the ```ebpfcore``` driver show up in the trace logs.
- 
- Apart from the calls to the tracing functions themselves, no new code was added. No new tests are required.

Resolves: #1565 

## Documentation

_Is there any documentation impact for this change?_
